README.md

```diff
@@ -40,26 +40,26 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [10.x]
+        node-version: [20.x]
```

Comment:
Even though CLI is built on Node@20, I think we should use here Node@18 because SPFx doesn't support Node@20 at the moment